src/process/process.rs

```diff
@@ -63,8 +63,9 @@ pub struct Callback {
     pub pc: usize
 }
 
-#[repr(C,packed)]
```

Comment:
Given that Tock is explicitly designed for Cortex Ms, I think it's OK to not worry about code being able to run on other architectures (except maybe Curie).